bson/src/test/unit/org/bson/types/DocumentSpecification.groovy

```diff
@@ -131,10 +131,10 @@ class DocumentSpecification extends Specification {
         Document document = Document.parse("{a: 1, b: {x: [2, 3, 4], y: {m: 'one', len: 3}}, 'a.b': 'two'}")
 
         then:
-        document.getEmbedded(List.of('notAKey'), String) == null
-        document.getEmbedded(List.of('b', 'y', 'notAKey'), String) == null
-        document.getEmbedded(List.of('b', 'b', 'm'), String) == null
-        Document.parse('{}').getEmbedded(List.of('a', 'b'), Integer) == null
+        document.getEmbedded(Arrays.asList('notAKey'), String) == null
```

Comment:
Should have mentioned this earlier.  Generally in Groovy tests we use Groovy's list syntax, which is just 

     ['a', 'b']

That will work on all versions

Reply:
I've made the change to use Groovy's list syntax.

Reply:
Evergreen Patch: https://evergreen.mongodb.com/version/5c549eae3066154949e671b5